pom.xml

```diff
@@ -163,6 +163,11 @@
         <cassandra.version>3.5.0</cassandra.version>
         <guava.version>20.0</guava.version>
         <tink.version>1.3.0-rc1</tink.version>
+        <commons-logging.version>1.1.3</commons-logging.version>
+        <commons-codec.version>1.10</commons-codec.version>
+        <snappy-java.version>1.1.7.2</snappy-java.version>
+        <antlr-runtime.version>3.5.2</antlr-runtime.version>
+        <joda-time.version>2.10.1</joda-time.version>
```

Comment:
I expect dependabot will update this once active in any case....

Reply:
Having looked at the janusgraph docs, I see we go with a default of berkeley (which makes sense), so is it correct that by adding the janusgraph components here we are adding in the required runtime jars so that we can do config such as 
```
storage.backend=cql
storage.hostname=localhost

index.search.backend=elasticsearch
index.search.hostname=100.100.101.1, 100.100.101.2
index.search.elasticsearch.client-only=true
```

I'm presuming We run Janusgraph locally , so the connector is more than a 'connector' .. its effectively the janus implementation too.

So on that basis I can see an argument for including these jars in as dependencies as they are tightly associated with janus and open up some additional configuration options - in this case for cassandra. Does this make sense to those reading? Anyone? ;-)

How about the apache components - can you explain how they are needed at runtime?